pkgs/applications/science/machine-learning/starspace/default.nix

```diff
@@ -0,0 +1,29 @@
+{lib, stdenv, fetchFromGitHub, boost165}:
+
+stdenv.mkDerivation rec {
+  pname = "starspace";
+  version = "20210117-${lib.strings.substring 0 7 rev}";
+  rev = "8aee0a950aa607c023e5c91cff518bec335b5df5";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "starspace";
+    inherit rev;
```

Comment:
```suggestion
    rev = "8aee0a950aa607c023e5c91cff518bec335b5df5";
```

Reply:
Applied in f8b69fa6d2fcf54548795e3e04845f304efb3a30